documentation/docs/00-introduction.md

```diff
@@ -14,7 +14,7 @@ SvelteKit is a framework for building extremely high-performance web apps.
 
 Building an app with all the modern best practices is fiendishly complicated. Those practices include [build optimizations](https://vitejs.dev/guide/features.html#build-optimizations), so that you load only the minimal required code; [offline support](#service-workers); [prefetching](#anchor-options-sveltekit-prefetch) pages before the user initiates navigation; and [configurable rendering](#ssr-and-javascript) that allows you to generate HTML [on the server](#ssr-and-javascript-ssr) or [in the browser](#ssr-and-javascript-router) at runtime or [at build-time](#ssr-and-javascript-prerender). SvelteKit does all the boring stuff for you so that you can get on with the creative part.
 
-It uses [Vite](https://vitejs.dev/) with a [Svelte plugin](https://github.com/sveltejs/vite-plugin-svelte) to provide a lightning-fast and feature-rich development experience, where changes to your code are reflected in the browser immediately.
+It uses [Vite](https://vitejs.dev/) with a [Svelte plugin](https://github.com/sveltejs/vite-plugin-svelte) to provide a lightning-fast and feature-rich development experience with [Hot Module Reload (HMR)](https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/config.md#hot), where changes to your code are reflected in the browser instantly.
```

Comment:
I think HMR stands for "Hot Module Replacement", though I've seen "Reload" being used interchangebly too. Regarding the linked URL, maybe we can link it to https://github.com/withastro/esm-hmr ?

Reply:
good catch!

I think linking there could be slightly confusing because it says Snowpack are the only ones using it: https://github.com/withastro/esm-hmr#whos-using-esm-hmr. perhaps if that gets updated then it'd make more sense to link to